.gitignore

```diff
@@ -1 +1,3 @@
 .idea/
+main
```

Comment:
Just curious on the background for adding these?

Reply:
I sometimes do a `go build main.go` which results in a `main` binary being added to the git diff. On the other hand, `web/harbour_bridge_output/` is generated when you run the frontend tests locally (we should fix the tests to cleanup though @Deep1998 FYI).